examples/dreamcast/2ndmix/Makefile

```diff
@@ -13,10 +13,10 @@ KOS_ROMDISK_DIR = romdisk_boot
 include $(KOS_BASE)/Makefile.rules
 
 clean: rm-elf
-	rm -f $(OBJS)
+	-rm -f $(OBJS)
```

Comment:
Not really an issue, but if you like you could use the `$(RM)` variable here (it expands to `rm -f`).

https://www.gnu.org/software/make/manual/html_node/Implicit-Variables.html#index-RM

Reply:
I think we want to keep -rm instead of rm.